core/src/main/java/com/tweet/core/model/Tweet.java

```diff
@@ -12,7 +9,10 @@
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long tweetId;
-    private String username;
+
+    @ManyToOne
+    @JoinColumn(name = "username")
+    private User username;
```

Comment:
Why not call this field user? username is just a User field. Specially gettrs and setters may be confusing, eq setUsername but you are setting user :)